README.md

```diff
@@ -4,30 +4,38 @@ The Java software Discourse Network Analyzer (DNA) is a qualitative content anal
 
 - Download the latest [release](https://github.com/leifeld/dna/releases) of the software.
 
-- Check out the detailed [manual](https://github.com/leifeld/dna/releases/download/v2.0-beta.22/dna-manual.pdf) for more information, including installation instructions and information on network methods and rDNA.
+- Check out the detailed [manual](https://github.com/leifeld/dna/releases/download/v2.0-beta.24/dna-manual.pdf) for more information, including installation instructions and information on network methods and `rDNA`.
 
 - If you have questions or want to report bugs, please create an issue in the [issue tracker](https://github.com/leifeld/dna/issues).
 
 <br />
 
 ## rDNA. A Package to Control DNA from R
 
-This is the companion package to DNA. It integrates the Java software with the statistical computing environment R.
+This is the companion package to DNA. It integrates the Java software with the statistical computing environment `R`.
+
+You can install the most recent release using:
 
-You can install the most recent version using:
 ``` r
-# install.packages("devtools")
-devtools::install_github("leifeld/dna/rDNA", INSTALL_opts = "--no-multiarch")
+# install.packages("remotes")
+remotes::install_github("leifeld/dna/rDNA@*release", INSTALL_opts = "--no-multiarch")
 ```
-Please note that the most recent version requires compilation of the current jar file from the sources on GitHub, for example using the provided make file.
 
-The recommended version for most users is therefore the stable release version of rDNA, which can be used together with the jar file of the respective release (see [releases](https://github.com/leifeld/dna/releases)). You can install the stable release version of rDNA by inserting the path to the latest release version into the above command. For example, for the 2.0 beta 23 release, this would be:
+This is the recommended version for most users. Note that the package relies on `rJava`, which needs to be installed first. For details on the installation process on different operating systems, consult the chapter "Installation of DNA and rDNA" in the [manual](https://github.com/leifeld/dna/releases/download/v2.0-beta.24/dna-manual.pdf).
```

Comment:
Please replace the quotation marks by LaTeX quotation marks.

Please also change all your edits such that there is one sentence per line.

Reply:
I'm not sure what you mean. The normal `` '' won't work in Markdown. Is there another way to get proper quotation marks?

Reply:
Oh, I should have paid more attention. I thought this was a change in the manual. Never mind. Also with regard to one sentence per line. We can merge if you can fix the error in the compilation.

Reply:
Ah okay, no problem! I think the last commit should fix it. The new code chunk tried to download the jar again, which travis really doesn't seem to like.